### PR TITLE
Add handler for wallet disconnect event

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

**Review by ShawTim — APPROVED** [review]

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx [hunk 1]
```diff
@@ -129,6 +129,42 @@ const Wallet = () => {
...
+  useEffect(() => {
+    if (!active) {
+      return;
+    }
+
+    const subscribeToAccountChanges = async () => {
+      const injectedConnector = new InjectedConnector({ supportedChainIds });
+      const provider = await injectedConnector.getProvider();
+
+      provider.on('accountsChanged', (accountsList: string[]) => {
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/wallet/Wallet.tsx:144]: we should also unsubscribe

**Review by danieledamiani — APPROVED** [review]

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:144]: I couldn't find method on provider that is used for unsubscribing, it seems it's automatically done as soon as wallet is disconnected/changed.

Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:144]: is `provider.off`not available?

Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:144]: No, I thought that something like that would be there, but I didn't find it.

**Pull request merged**